src/index.ts

```diff
@@ -34,7 +43,16 @@ const manager: JupyterFrontEndPlugin<ISecretsManager> = {
     connector: ISecretsConnector
   ): ISecretsManager => {
     console.log('JupyterLab extension jupyter-secrets-manager is activated!');
-    return new SecretsManager({ connector });
+    const secretsManager = new SecretsManager({ connector });
+    const panel = new Panel();
+    panel.id = 'jupyter-secrets-manager:panel';
+    panel.title.icon = logoIcon;
```

Comment:
There is already a lock icon in [Jupyterlab](https://github.com/jupyterlab/jupyterlab/blob/main/packages/ui-components/style/icons/lock.svg), we should reuse it.

```ts
import { lockIcon } from '@jupyterlab/ui-components';
```

Reply:
good catch, I'll change to reuse that.